exec/java-exec/src/main/java/org/apache/drill/exec/rpc/data/DataClientConnection.java

```diff
@@ -84,8 +85,8 @@ public boolean equals(Object obj) {
     return true;
   }
 
-  public void shutdownIfClient() {
-    Closeables.closeQuietly(client);
+  public void shutdownIfClient() throws IOException {
```

Comment:
No need for throws IOException, Closeables.close(..., true) makes sure it's not propagated